english/marshmallow.md

```diff
@@ -732,4 +732,5 @@ Instead of reading this entire story, you decide to go see the new star wars!
 
 You decide that $1.5 billion is too much to pass up, so you buy a [Powerball ticket](powerball/powerball.md).
 
-[Do you have my stapler?](stapler/stapler.md)
\ No newline at end of file
+[Do you have my stapler?](stapler/stapler.md)
+Join the new Milk Cafe where you can listen to Debussy all day and sip flavoured milks! [Milk Cafe this way!](milkcafe/milkcafe.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.